src/main/kotlin/net/leanix/githubagent/services/SyncLogService.kt

```diff
@@ -13,14 +13,22 @@ class SyncLogService(
     private val cachingService: CachingService
 ) {
     fun sendErrorLog(message: String) {
-        sendSyncLog(message, LOGS_TOPIC, null, LogLevel.ERROR)
+        sendSyncLog(message, LOGS_TOPIC, Trigger.PROGRESS_FULL_SYNC, LogLevel.ERROR)
```

Comment:
I think won't work. as this will fail:
 val runId = cachingService.get("runId") as UUID